src/utils/getData.js

```diff
@@ -0,0 +1 @@
+const hola = hola;

```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'hola' was used before it was defined. [no-use-before-define](https://eslint.org/docs/rules/no-use-before-define)